rfc/0084-remove-0081-active-user-beacon.md

```diff
@@ -0,0 +1,42 @@
+# Revert RFC 81: Active User Beacon
+
+## Summary
+
+This RFC reverts the RFC 81: active user beacon.
+
+## Motivation
+
+A significant number of developers and users have expressed unhappiness at the current opt-out behaviour of the active user beacon, and want it changed or removed. Since not annoying users is important, I'd like to remove the beacon entirely.
+
+Some people have made the argument that other projects have similar amounts (or significantly more) user tracking. I don't think this justifies this inclusion - each case of tracking should be looked at separately.
+
+## Explanation
+
+This requires three changes:
+
+- The `specification/0081-active-user-beacon.md` file should be removed from this repository.
+- The `ActiveUserBeacon.java` file in the quilt-loader repository should be removed, as well as any references to it in the repository.
+- The infrastructure team should take down the server at `beacon.quiltmc.org`.
+
+
+## Drawbacks
+
+This potentially means less sponsorships for quilt in the future, which is not ideal.
+
+
+## Rationale and Alternatives
+
+In theory we could change the beacon to be opt-in, rather than opt-out. However this has the disadvantage of making the "monthly active users" metric useless, which renders the whole feature pointless.
+
+
+## Prior Art
+
+Many other projects have telementry (of various kinds) and many have had objectors.
+
+For example [audacity](https://en.wikipedia.org/wiki/Audacity_audio_editor#Reception) tried to add (much) more invasive telementry, and then removed it after much user backlash. This example isn't fully applicable to quilt though, since we don't collect anywhere near as much user data of them.
+
+## Unresolved Questions
+
+Does this prevent us from having any sort of opt-in telementry in the future? For example, an update checker for quilt-loader - would people be okay with an option for quilt-loader to check quilts servers for updates if a mod requires a newer version of loader?
+Or should that go in a future RFC?
```

Comment:
No, this would not prevent implementing telemetry in the future, but yes, it should have another RFC when that is suggested.

And in that new RFC, in my opinion these criteria should be met:

- Practical use cases, aka quotes from parties who _need_ this sort of telemetry before implementing and saying "this might come useful in the future"
- More technical audits to ensure this would do only what it is supposed to, to prevent [this from happening again](https://github.com/QuiltMC/quilt-loader/issues/334#issuecomment-1638083180)
- More detailed (maybe even separate) discussions of 
   - what the telemetry should do
   - how it should do it
   - opt-in/opt-out methods
- Involve relevant parties (launchers, users, modpacks) for opt-in/opt-out method discussion, to ensure everyone can be satisfied in indicating their preference 
- More user involvement and awareness overall _before RFC is merged_

Reply:
I'm open to such an RFC in the future. I'm not against telemetry 100% all of the time, and if such discussions were to occur I'd want to be a part of them. We could absolutely come back with more of an emphasis on communicating to users, articulating some value that gets returned to them, and giving them control, and come up with completely different results.

I also want to say while I'm not convinced by the privacy characteristics of the beacon, I'm also not completely convinced the beacon was collecting IPs. Could be an idle threat, or more likely ordinary connection logging / fail2ban type things happening by the hoster was leaking them. Kind of a moot point, as the optics are *super bad*, but yeah. Let's talk about this next time.